packages/app-project/src/components/ZooHeaderWrapper/ZooHeaderWrapperContainer.js

```diff
@@ -79,11 +79,19 @@ class ZooHeaderWrapperContainer extends Component {
       })
   }
 
+  unreadMessages () {
+    const { user } = this.props
+
+    return (user.isLoggedIn && user.personalization.notifications.unreadConversationsIds.length)
+      ? user.personalization.notifications.unreadConversationsIds.length
+      : 0
```

Comment:
`user.personalization.notifications.unreadConversationsIds.length` is 0 by default, which would simplify the logic here. At the moment, this block is the equivalent of:
```js
if (user.personalization.notifications.unreadConversationsIds.length === 0) {
  return 0
}
return user.personalization.notifications.unreadConversationsIds.length
```

Which is the same as just returning `user.personalization.notifications.unreadConversationsIds.length`.